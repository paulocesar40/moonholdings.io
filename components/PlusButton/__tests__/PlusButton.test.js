import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import PlusButton from '../PlusButton'

describe('<PlusButton /> component', () => {
  describe('rendering', () => {
    const wrapper = shallow(<PlusButton toggleSearch={jest.fn()} />);
    it('should render a component matching the snapshot', () => {
      const tree = toJson(wrapper);
      expect(tree).toMatchSnapshot();
      expect(wrapper).toHaveLength(1);
    });
  });
});
