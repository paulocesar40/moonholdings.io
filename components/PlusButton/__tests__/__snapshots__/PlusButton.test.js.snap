// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PlusButton /> component rendering should render a component matching the snapshot 1`] = `
<plusButton__PlusButtonStyle
  onClick={[MockFunction]}
  tabIndex={0}
>
  <span>
    +
  </span>
  <h1>
    Add Asset
  </h1>
</plusButton__PlusButtonStyle>
`;
